masterlist.yaml

```diff
@@ -2514,10 +2514,6 @@ plugins:
     clean:
       - crc: 0xFC447339
         util: TES5Edit v3.2.2
-  - name: 'The Eyes Of Beauty - Elves Edition.esp'
-    clean:
-      - crc: 0xAFF2DA5E
-        util: TES5Edit v3.2.2
```

Comment:
I also wouldn't consider just clean data enough to add an entry.
If there was more to go along with it such as dirty, bash tags, load after or group changes then it would be fine.

Reply:
I don't understand your reasoning here. Does clean data have a downside I'm not aware of, e.g. it's a very slow function? Personally I consider clean data LOOT's most valuable asset. There is no way to know if a mod needs cleaning without cleaning it first. This is a staggering process for large load orders and error prone:
- Did I already clean that mod?
- Did that mod I reinstalled to pick a different option contain dirty edits and I have to clain it again?
- Nice I cleaned another mod, oh dammit was that the one I'm not supposed to clean?
- That mod again? I think I cleaned it already but let's just check again for the fourth time to be sure...

I tried keeping notes of this stuff, but it doesn't really solve the problem. The only proper solution is a database that checks for checksums like LOOT (or just give up and hope none of the installed mods will cause actual damage).

Reply:
@Kaenguru-Knobel-Kette Because it's more efficient to mark dirty plug-ins. We can't add every clean plug-in, Nexus has 50,000+ mod pages, how many of those have plug-ins? How many of them have multiple patches?

Reply:
> We can't add every clean plug-in

We can't add every dirty plugin either - for every clean state there are many possible dirty states, after all. I think if @Kaenguru-Knobel-Kette is supplying us with the information, it's worth keeping. I added the feature because people said they'd find it useful, and as far as I'm aware, people do.

If you're concerned about the masterlist getting filled up with `clean` entries or just too large to edit effectively, let's cross that bridge when we come to it. There are a bunch of options for dealing with that, and we're best off picking the option that fits the situation we find outselves in, when we're in it.

Reply:
@CptMcSplody Should I open a new pull request or will you commit them directly?

Reply:
@Kaenguru-Knobel-Kette  Open a new pull. You did the work after all.